web/impact/impact/v1/views/organization_list_view.py

```diff
@@ -13,6 +13,13 @@ class OrganizationListView(BaseListView):
     view_name = "organization"
     helper_class = OrganizationHelper
 
+    def filter(self, queryset):
```

Comment:
Not a blocker, but it would be nice to push this down into BaseListView so any object that has a .name automatically gets the ability to search by name substring.

Reply:
@mc-nathan done.